automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/analysis/MethodLaunchSpec.scala

```diff
@@ -164,8 +164,8 @@ class MethodLaunchSpec extends FreeSpec with ParallelTestExecution with Matchers
               eventually {
                 val status = submissionDetailsPage.getApiSubmissionStatus(billingProject, workspaceName, submissionId)
                 logger.info(s"Status is $status in Submission $billingProject/$workspaceName/$submissionId")
-                withClue(s"Monitoring Submission $billingProject/$workspaceName/$submissionId. Status is Done or Running.") {
-                  status should (equal("Done") or equal ("Running"))
+                withClue(s"Monitoring Submission $billingProject/$workspaceName/$submissionId. Status is Done or Submitted.") {
```

Comment:
(optional) since, quite evidently, this has been a point of confusion, maybe we could use a comment pointing out that a submission status of submitted corresponds to a workflow state of running